utils/test_harness/tools/src/test_harness_metric.cpp

```diff
@@ -516,8 +516,8 @@ metric_streamer_open(zet_metric_group_handle_t matchedGroupHandle,
 
 zet_metric_streamer_handle_t metric_streamer_open_for_device(
```

Comment:
why two definitions of metric_streamer_open_for_device () ?

Reply:
There is just the 1 definition. I have just made  `notifyEveryNReports` and `samplingPeriod` as reference so when MDAPI changes them they are reflected back in the scope of the tests to more accurately predict the time at which the events will be generated.

Reply:
thanks for the clarification